clang/lib/Driver/ToolChains/BareMetal.h

```diff
@@ -78,8 +86,9 @@ class LLVM_LIBRARY_VISIBILITY BareMetal : public ToolChain {
   using OrderedMultilibs =
       llvm::iterator_range<llvm::SmallVector<Multilib>::const_reverse_iterator>;
   OrderedMultilibs getOrderedMultilibs() const;
-
+  bool UseLD;
```

Comment:
I know of at least one toolchain that derives from BareMetal and defaults to LLD.  Please make this a protected member so it can be overwritten to FALSE.  See my other comment on the line in the constructor where UseLD value is being calculated.

Reply:
please see my comment below